bin/reth/src/node/events.rs

```diff
@@ -26,27 +28,30 @@ const INFO_MESSAGE_INTERVAL: Duration = Duration::from_secs(25);
 
 /// The current high-level state of the node.
 struct NodeState {
+    db: Arc<DatabaseEnv>,
```

Comment:
needs doc that this is used for db metrics included in logs

Reply:
added